test/unit/staging-helpers.ts

```diff
@@ -27,7 +27,7 @@ export function bootstrapTestEnvironment(
 
         return instance.setup(Object.assign({
           backend: {                                  // <-- configure backend settings
-            loadPath: "./locales/{{lng}}/{{ns}}.json", // <-- XHR settings for where to get the files from
+            loadPath: "./fixtures/locales/{{lng}}/{{ns}}.json", // <-- XHR settings for where to get the files from
```

Comment:
why adding the `fixtures` folder? This is anyway proxied by the moduleNameMapper entry in package.json for the locales.

Reply:
see https://github.com/aurelia/i18n/pull/288#discussion_r232405703